test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatPushTests.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NuGet.CommandLine.XPlat;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    public class XPlatPushTests
+    {
+        [Theory]
+        [InlineData(TestServers.MyGet, nameof(TestServers.MyGet), false)]
+        [InlineData(TestServers.ProGet, nameof(TestServers.ProGet), false)]
+        [InlineData(TestServers.Nexus, nameof(TestServers.Nexus), true)]
+        // [InlineData(TestServers.Artifactory, nameof(TestServers.Artifactory), false)] // Trial license expired!
+        // [InlineData(TestServers.Klondike, nameof(TestServers.Klondike), false)] // 500 Internal Server Error pushing
+        // [InlineData(TestServers.NuGetServer, nameof(TestServers.NuGetServer), false)] // doesn't work (credentials?)
```

Comment:
Credentials weren't being used, so I fixed that.
But now this fails with "500 internal server error" because the server doesn't think that the package contains a manifest. Although it does. Could this be a server error?

---
In reply to: [68082931](https://github.com/NuGet/NuGet.Client/pull/706#discussion_r68082931) [](ancestors = 68082931)